front/lib/api/assistant/jit_actions.ts

```diff
@@ -55,6 +58,58 @@ export async function isJITActionsEnabled(
   return use;
 }
 
+export function isConversationQueryableFileContentType(
+  contentType: SupportedContentFragmentType
+): boolean {
+  if (isSupportedImageContentType(contentType)) {
+    return false;
+  }
+  // For now we only allow including text files.
+  switch (contentType) {
+    case "application/msword":
+    case "application/vnd.openxmlformats-officedocument.wordprocessingml.document":
+    case "application/pdf":
+    case "text/markdown":
+    case "text/plain":
+    case "dust-application/slack":
+    case "text/tab-separated-values":
```

Comment:
why not true?

Reply:
(for now) We do not support it in the upsert, only csv.
I don't know why and I wanted to check that later (as well as adding more file format support such as xls)

Reply:
ack